pkg/options/options.go

```diff
@@ -111,31 +111,31 @@ func (o Options) Validate() error {
 
 // SubOptions allow user to overwrite certain options at path/operation level using x-kusk extension
 type SubOptions struct {
-	Disabled *bool `yaml:"disabled,omitempty" json:"disabled,omitempty"`
+	Hidden *bool `yaml:"hidden,omitempty" json:"hidden,omitempty"`
 	// Upstream is a set of options of a target service to receive traffic.
 	Upstream *UpstreamOptions `yaml:"upstream,omitempty" json:"upstream,omitempty"`
 	// Redirect specifies thre redirect optins, mutually exclusive with Upstream
 	Redirect *RedirectOptions `yaml:"redirect,omitempty" json:"redirect,omitempty"`
 	// Path is a set of options to configure service endpoints paths.
-	Path        *PathOptions       `yaml:"path,omitempty" json:"path,omitempty"`
-	QoS         *QoSOptions        `yaml:"qos,omitempty" json:"qos,omitempty"`
-	CORS        *CORSOptions       `yaml:"cors,omitempty" json:"cors,omitempty"`
-	Websocket   *bool              `json:"websocket,omitempty" yaml:"websocket,omitempty"`
-	Validation  *ValidationOptions `json:"validation,omitempty" yaml:"validation,omitempty"`
-	Mocking     *MockingOptions    `json:"mocking,omitempty" yaml:"mocking,omitempty"`
-	RateLimit   *RateLimitOptions  `json:"rate_limit,omitempty" yaml:"rate_limit,omitempty"`
-	Cache       *CacheOptions      `json:"cache,omitempty" yaml:"cache,omitempty"`
-	OpenAPIPath string             `json:"openapi-path,omitempty" yaml:"openapi-path,omitempty"`
-	Auth        *AuthOptions       `json:"auth,omitempty" yaml:"auth,omitempty"`
+	Path          *PathOptions       `yaml:"path,omitempty" json:"path,omitempty"`
+	QoS           *QoSOptions        `yaml:"qos,omitempty" json:"qos,omitempty"`
+	CORS          *CORSOptions       `yaml:"cors,omitempty" json:"cors,omitempty"`
+	Websocket     *bool              `json:"websocket,omitempty" yaml:"websocket,omitempty"`
+	Validation    *ValidationOptions `json:"validation,omitempty" yaml:"validation,omitempty"`
+	Mocking       *MockingOptions    `json:"mocking,omitempty" yaml:"mocking,omitempty"`
+	RateLimit     *RateLimitOptions  `json:"rate_limit,omitempty" yaml:"rate_limit,omitempty"`
+	Cache         *CacheOptions      `json:"cache,omitempty" yaml:"cache,omitempty"`
+	PublicAPIPath string             `json:"public_api_path,omitempty" yaml:"public-api-path,omitempty"`
```

Comment:
`public_api_path` vs `public-api-path`.

I would use `public-api-path`.

Reply:
underscore is so it is easer to mark the string when copying

Reply:
so we are going with `public_api_path` instead of `public-api-path` because the former is easier to select in the editor where words separated with hyphen require more than 3 clicks to select